web_scraping/epic_games_scraper/lambda_handler.py

```diff
@@ -0,0 +1,14 @@
+"""Script that runs when AWS Lambda is triggered."""
+
+from extract_epic import process_listings
+
+# pylint: disable=W0613
```

Comment:
What did you mean by this

Reply:
Unused variable, lambda handlers need the two variables